projects/storefrontlib/src/lib/my-account/order/order-history/order-history.component.spec.ts

```diff
@@ -13,13 +13,14 @@ import { UserService } from '../../../user/facade/user.service';
 import { PaginationAndSortingModule } from '../../../ui/components/pagination-and-sorting/pagination-and-sorting.module';
 
 import { OrderHistoryComponent } from './order-history.component';
+import { UserOrders } from '../../models/order.model';
 
-const mockOrders = {
+const mockOrders: UserOrders = {
   orders: [
     { code: 1, placed: 1, statusDisplay: 'test', total: { formattedValue: 1 } }
   ],
   pagination: { totalResults: 1, sort: 'byDate' },
-  sorts: [{ code: 'byDate', selected: true }]
+  sort: [{ code: 'byDate', selected: true }]
 };
 
```

Comment:
Can we add types on lines 30, 31, 32, 35, 36, 39, 66 and 76?

Reply:
for mockAuthService, mockRoutingService, mockUserService, (30,31,32) I don't know how to type them. If you do, let me know :)

Reply:
Instead of `useValue` try with `useClass`

Reply:
Causes 10 errors... :/

Reply:
Thanks man